plugins/env.json

```diff
@@ -0,0 +1,9 @@
+{
+  "apiKey": "AIzaSyBLtesrs_qV1jdX4yTSVvqmv_f65JPzolE",
```

Comment:
@priyankashrma We should not expose any sort of secret keys, because any on can use your key to manipulate the data. create a .env file in your project folder and access from that 
eg : FIREBASE_API_KEY="xxxxxxxxxxxxxxxxxx" and you can access that like apiKey: process.env.FIREBASE_API_KEY,